ci/linux-gsc-ubuntu18.04-release.jenkinsfile

```diff
@@ -1,20 +1,20 @@
 node(node_label) {
     env.ORIG_WKSP = env.WORKSPACE
-    def build_ok = true
+    env.build_ok = true
     try {
         stage('checkout'){
             dir ('./') {
                 git url: 'https://github.com/jinengandhi-intel/graphene_local_ci.git'
             }
 
-            dir('gramine') {
-                git url: 'https://github.com/gramineproject/gramine.git'
+            dir('gsc_dev_repo') {
```

Comment:
Our repo also has a folder called gsc which conflict with the repo name gsc, so checking out in a different folder.